drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/converters/column/impl/ActionSetFieldColumnConverter.java

```diff
@@ -75,8 +75,18 @@ public boolean handles( final BaseColumn column ) {
     @Override
     public List<GridColumn.HeaderMetaData> makeHeaderMetaData( final BaseColumn column ) {
         return new ArrayList<GridColumn.HeaderMetaData>() {{
-            add( new BaseHeaderMetaData( column.getHeader(),
-                                         ActionCol52.class.getName() ) );
+            if ( column instanceof ActionSetFieldCol52 ) {
+                ActionSetFieldCol52 actionSetFieldColumn = ( ActionSetFieldCol52 ) column;
+                if ( actionSetFieldColumn.getBoundName() != null && !actionSetFieldColumn.getBoundName().isEmpty() ) {
```

Comment:
This should never be false; but if it is you probably want to include the second-level ```BaseHeaderMetaData``` entry within the ```if(..) {..}``` block.